share/revng/test/tests/analysis/ImportFromCAnalysis/enum-without-an-annotation/expected-error.txt

```diff
@@ -0,0 +1,7 @@
+#
+# This file is distributed under the MIT License. See LICENSE.md for details.
+#
+
+import-from-c: Unable to parse the enum annotation.
```

Comment:
What's the `enum annotation`?
What about `enum with invalid underlying type: unable to parse _ENUM_UNDERLYING annotation`?

Reply:
Done.